diagrams/py/prod/sandbox_pipeline_design.py

```
@@ -0,0 +1,40 @@
+from diagrams import Cluster, Diagram, Edge
```

Ruff (0.7.0)

`diagrams.Edge` imported but unused

Remove unused import: `diagrams.Edge`

(F401)